src/azul/logging.py

```diff
@@ -133,6 +136,22 @@ def silent_es_log_level():
 
 
 es_log = logging.getLogger('elasticsearch')
+aws_log = logging.getLogger('aws')
+
+
+def http_body_log_message(log: logging.Logger,
+                          body_type: str,
+                          body: bytes | bytearray | str | None
+                          ) -> str:
+    if body is None:
+        return f'… without {body_type} body'
+    else:
+        if log.isEnabledFor(logging.DEBUG):
```

Comment:
@hannes-ucsc, should we be concerned about [Boto3 logs printing SecretString when log level is set to debug
#2292](https://www.github.com/boto/boto3/issues/2292#issuecomment-586442819)?

If we ever use boto3 secretsmanager or KMS clients, with aws logging enabled at DEBUG, might this be an issue?

Reply:
Good question. Yes, if/when we want to enable it for those clients, we need to take care of that.

Next time you run the IT for this in your personal deployment, please request a PL slot for both of us to review the logs emitted by that IT in CW. And please add a checklist item for me to review the job logs on GitLab once the sandbox builds are complete.